config/bootstrap.php

```diff
@@ -233,8 +234,15 @@
     ['bootstrap' => true, 'routes' => true, 'ignoreMissing' => true]
 );
 
-/**
- * Load common plugins like 'Migrations' and 'DebugKit' or other custom / 3rd party plugins
- * via configuration key 'Plugins'
+if (Configure::read('debug')) {
```

Comment:
I would prefer to decide in `app.php` if I want to load the plugin.

```php
'Plugins' => [
        'BEdita/DevTools' => ['debugOnly' => true, 'bootstrap' => true],
]
```

In this way I could activate it just if I need it, optionally also with debug off.